src/service/get-service.js

```diff
@@ -148,6 +120,31 @@ class GetService extends OperationService {
                     tokenId,
                     keyword,
                 );
+                if (paranetNodesAccessPolicy === 'CURATED' && responseData.privateNquads) {
+                    await this.tripleStoreService.localStoreAsset(
+                        repository,
+                        responseData.syncedAssetRecord.privateAssertionId,
+                        responseData.privateNquads,
+                        blockchain,
+                        contract,
+                        tokenId,
+                        keyword,
+                    );
+                }
+                const privateAssertionId =
+                    paranetNodesAccessPolicy === 'CURATED'
```

Comment:
Do we even need this check? It's either in the data or 'undefined'

Reply:
Didnt' want to risk it